nbgitpuller/handlers.py

```diff
@@ -38,6 +40,53 @@ def emit(self, data):
         self.write('data: {}\n\n'.format(serialized_data))
         yield self.flush()
 
+    def setup_plugins(self, provider):
+        pm = pluggy.PluginManager("nbgitpuller")
+        pm.add_hookspecs(hookspecs)
+        pm.load_setuptools_entrypoints("nbgitpuller", name=provider)
+        return pm
+
+<<<<<<< HEAD
```

Comment:
there seem to be some merge markers left in - am not sure if the code here should be kept or not?

Reply:
Ahhh... I blew it here. I merged my branch off this feature branch back-in and then did not run the tests again. Didn't even think about it. I will fix this and get it back in.  Should I rebase against the main branch now or wait until we have this solid?